src/secure_random.cr

```diff
@@ -1,19 +1,66 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others. It is implemented with `OpenSSL` as well as 
+# `LibCrypto` libraries.
+#
+# For example:
+# ```crystal
+# SecureRandom.base64 #=> LIa9s/zWzJx49m/9zDX+VQ==
+# SecureRandom.hex    #=> c8353864ff9764a39ef74983ec0d4a38
+# SecureRandom.uuid   #=> c7ee4add-207f-411a-97b7-0d22788566d6
+# ```
 module SecureRandom
+  # Generates a random base64 string.
+  #
+  # The argument `n` determines the number of generated bytes.
+  # The bytes are encoded with base64 and will slightly increase in size.
+  # A base64 string may contain A-Z, a-z, 0-9, +, /, and =.
+  # Check `Base64#strict_encode` for details.
```

Comment:
The description is so detailed, it makes the reference entirely redundant. Yet I really do want the reference, I don't get why you want to duplicate the description here. Base64 is well known, if you search for it on SecureRandom you know what you want anyway.